### ISSUE TITLE
A more detail and complex way to enter a job offer description

### ISSUE THREAD

**Issue by Theryx** [osscameroon/jobsika#452]
Currently companies can only enter plain text into the job offer detail input field. We need to give them the possiblity to enter elements like: 

- Bold
- Itallics
- Underline
- Tittles
- etc

**Comment by jordyparker** [comment]
Hi @Theryx @Zaker237 

I'd like to work on this feature. Can I go ahead ?

**Comment by Zaker237** [comment]
hallo @jordyparker go ahead and tag me on it

**Comment by jordyparker** [comment]
can I use a WYSIWYG editor for the case ?

**Comment by Zaker237** [comment]
yes that was exactly the plan. wirting the new reusable WYSIWYG component for that

**Comment by jordyparker** [comment]
can you have a look at this [editor](https://www.vue2editor.com) ? 
Or I should go from scratch ?

**Comment by Zaker237** [comment]
@Asam237 any preference??

@jordyparker for me you ca feel free

**Comment by jordyparker** [comment]
Hi @Zaker237 please have a look at PR #455
